src/stdlib/pervasives.md

```diff
@@ -68,6 +68,22 @@ data Array<a>
 
 The type of Grain arrays, i.e. `[> 1, 2, 3]`.
 
+## Comparison Operations
+
+Operations to compare values.
+
+### Pervasives.**(==)**
+
+Check that two values are equal. This checks for structural equality, so it also works for comparing things like tuples and lists.
+
+### Pervasives.**(!=)**
+
+Check that two values are **not** equal. This checks for structural equality, so it also works for comparing things like tuples and lists.
+
+### Pervasives.**(eq)**
+
+Checks that two values are physically equal. Use this operator if you don't need or want structural equality.
```

Comment:
My bad for not catching this in the other PR. We already have this, and it's like Python's `is`.

```suggestion
### Pervasives.**(is)**

Checks that two values are physically equal. Use this operator if you don't need or want structural equality.
```

Reply:
Hah, I was staring directly at the `as` method and typed `eq`. My bad and good catch.